apps/hodejegeren/config.yml

```diff
@@ -42,6 +42,12 @@ spec:
           cluster: dev-fss
         - application: mn-synt-arbeidsforhold-service
           cluster: dev-fss
+        - application: testnav-oversikt-frontend
+          cluster: dev-gcp
+    outbound:
+      rules:
+        - application: testnav-oversikt-frontend
+          cluster: dev-gcp
```

Comment:
Var litt dårlig forklart av meg, men testnav-oversikt-frontend skal ikke være i outbound rules. Mente de som hodejegeren har consumer for, altså `tps-forvalteren-proxy` :)

Reply:
fikser